### DISCUSSION TITLE
Support for LDAP Authentication

### DISCUSSION THREAD

**Discussion by andresC98** [langfuse/langfuse#4837] [Ideas]
### Describe the feature or potential improvement

Given that NextAuth supports many different Auth/Credentials provider, Langfuse could [add support](https://langfuse.com/self-hosting/authentication-and-sso) for LDAP Authentication.

[Next-Auth LDAP Auth Example Integration](https://next-auth.js.org/tutorials/ldap-auth-example)

**Comment by marcklingen** [comment]
Hi @andresC98, open for contributions on this if you are interested!

**Reply by marcklingen** [reply]
> [Next-Auth LDAP Auth Example Integration](https://next-auth.js.org/tutorials/ldap-auth-example)

Just had a look into this. The `ldapjs` package that this guide recommends has been decommissioned ([npm](https://www.npmjs.com/package/ldapjs)) since then. I don't think this is a good route to take here and I'd recommend to use an ldap<>oidc bridge to authenticate via ldap.
